classes/pdf/HTMLTemplate.php

```diff
@@ -101,9 +101,9 @@ protected function getLogo()
         $id_shop = (int) $this->shop->id;
 
         if (Configuration::get('PS_LOGO_INVOICE', null, null, $id_shop) != false && file_exists(_PS_IMG_DIR_ . Configuration::get('PS_LOGO_INVOICE', null, null, $id_shop))) {
-            $logo = _PS_IMG_DIR_ . Configuration::get('PS_LOGO_INVOICE', null, null, $id_shop);
+            $logo = _PS_IMG_ . Configuration::get('PS_LOGO_INVOICE', null, null, $id_shop);
         } elseif (Configuration::get('PS_LOGO', null, null, $id_shop) != false && file_exists(_PS_IMG_DIR_ . Configuration::get('PS_LOGO', null, null, $id_shop))) {
-            $logo = _PS_IMG_DIR_ . Configuration::get('PS_LOGO', null, null, $id_shop);
+            $logo = _PS_IMG_ . Configuration::get('PS_LOGO', null, null, $id_shop);
         }
 
         return $logo;
```

Comment:
The logo is generated from an URL. In consequence we can send the proper address to the template.

```suggestion
         return Tools::getShopProtocol() . Tools::getMediaServer(_PS_IMG_) . $logo;
```